sdk/eventhub/Azure.Messaging.EventHubs/src/EventHubProducer.cs

```diff
@@ -344,6 +373,26 @@ protected EventHubProducer()
         [EditorBrowsable(EditorBrowsableState.Never)]
         public override string ToString() => base.ToString();
 
+        private DiagnosticScope CreateDiagnosticScope()
+        {
+            DiagnosticScope scope = _clientDiagnostics.CreateScope("Azure.Messaging.EventHubs.EventHubProducer.Send");
```

Comment:
Should we consider `nameof` for this or do we want this to remain static even if we refactor?